### ISSUE TITLE
bc is not installed default arch based distros

### ISSUE THREAD

**Issue by kerberos272** [TuxVinyards/qqX#8]
qqx acting weird and get errors on arch because of  "bc"  is not installed  by default.

installing it solve problems.
Maybe  warning in installer could be useful

**Comment by TuxVinyards** [comment]
@kerberos272  thanks for that  :+1: 

Seems to be something some that crept in recently.  I noticed it when testing on Garuda Dr460nized  a few days ago too. Same with Manjaro KDE minimal.  Kubuntu 24.04 minimal, the latest version which has KDE 5, has bc  though.

There's a checker that's now been added to 1.9.04  and I put some notes in the installer wiki  the other day too.

`bc` is one those things that I always thought to be in  coreutils or util-linux  but  looking more closely I found that it is only seems to be part of 'Unix SUS2008 utilities'  

Perhaps it will start getting added to Bash packages, in the future, as a recommended?

Have a look at 1.9.04.  It's shaping up quite nicely and has some got some good new features.

See https://github.com/TuxVinyards/qqX?tab=readme-ov-file#release-notes
